components/ContactCard/ContactCard.module.scss

```diff
@@ -64,12 +64,17 @@
 }
 
 .linkText {
+    @include text-label;
+    @include link-primary;
+
     overflow: hidden;
     text-overflow: ellipsis;
     white-space: nowrap;
 }
 
 .icon {
+    @include link-primary;
```

Comment:
Why do you need to apply the `link-primary` mixin on the `linkText` and `icon` again? It's on the `link` itself so it should work? 🤔 

If it does not then you should be just able to do `color: inherit`, right?